dagger-common/build.gradle

```diff
@@ -80,6 +80,8 @@ dependencies {
     testImplementation 'com.google.protobuf:protobuf-java:3.5.0'
     testImplementation 'com.google.protobuf:protobuf-java-util:3.5.0'
     testImplementation 'org.grpcmock:grpcmock-junit5:0.5.0'
+    testImplementation 'org.powermock:powermock-module-junit4:2.0.0-beta.5'
+    testImplementation 'org.powermock:powermock-api-mockito2:2.0.0-beta.5'
```

Comment:
let's not use powermock if possible, it messes up the classpaths

Reply:
Removed